spec/dummy/app/resources/article_resource.rb

```diff
@@ -24,4 +24,8 @@ def id=(external_id)
   # Setting this attribute is an easy way to test that authorizations work even
   # when the model has validation errors
   attributes :blank_value
+
+  def self.creatable_fields(context)
+    super + [:id]
+  end
```

Comment:
Thanks @thibaudgg! Let us know if you hit roadblocks.

These might be relevant to your tests, if you haven't updated in a while: #53 #40